### DISCUSSION TITLE
能否清空访客统计数据

### DISCUSSION THREAD

**Discussion by Moriefy** [Qexo/Qexo#586] [Q&A] [answered]
之前用好多个ip来访问博客，以至于访问量数据太大了，不太真实，所以想请问一下有没有办法清空访客统计的数据。

**Comment by am-abudu** [comment]
我应该稍后会推送一个云端命令用于这一功能

**Reply by Moriefy** [reply]
期待！

**Comment by am-abudu** [comment, marked as answer]
已推送 [5aa2151](https://github.com/Qexo/Scripts/commit/5aa2151225a078951eb10626eb1c29fa3ad5c2c2) 请在设置->云端命令库查看
